app/src/main/kotlin/com/hippo/ehviewer/ui/screen/SearchBarScreen.kt

```diff
@@ -239,51 +243,66 @@ fun SearchBarScreen(
                     .padding(top = 48.dp) then scrollAwayModifier,
             )
         }
+        val onActiveChange = { v: Boolean ->
```

Comment:
nit: Rename to expended & onExpandedChange

Reply:
Done